locale/emoji-buckets.pot

```diff
@@ -0,0 +1,62 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-04-15 21:14+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: emoji_menu.js:34
+msgid "Smileys & People"
+msgstr ""
+
+#: emoji_menu.js:35
+msgid "Nature"
+msgstr ""
+
+#: emoji_menu.js:36
+msgid "Food"
+msgstr ""
+
+#: emoji_menu.js:37
+msgid "Activities"
+msgstr ""
+
+#: emoji_menu.js:38
+msgid "Travel"
+msgstr ""
+
+#: emoji_menu.js:39
+msgid "Objects"
+msgstr ""
+
+#: emoji_menu.js:40
+msgid "Symbols"
+msgstr ""
+
+#: emoji_menu.js:41
+msgid "Flags"
+msgstr ""
+
+#: emoji_menu.js:74
+msgid "Preferences..."
+msgstr ""
+
+#: prefs.js:26
+msgid "Font size"
+msgstr ""
+
+#: prefs.js:26
+msgid "requires restart"
+msgstr ""
```

Comment:
I'm not sure why we need this file along with `emoji-buckets.po`

Reply:
It is the starting point for new translations. Run ```xgettext -k_ -kN_ -o locale/emoji-buckets.pot *.js --from-code=UTF-8``` to update it automatically from the code. There are developers separating po files (sources) from mo files (binaries) and provide a makefile or a script to build the translation (create the mo files) automatically. See [Dash to Panel](https://github.com/home-sweet-gnome/dash-to-panel) e.g. But even they check in the pot file for translators convenience.